src/components/LayoutTree.tsx

```diff
@@ -111,6 +114,31 @@ const useHideSpaceAbove = (spaceAbove: number) => {
   )
 }
 
+/** Calculate if the cursor is greater than the previous sibling at LayoutTree level. */
+const useCursorIsGreaterThanPrevSibling = (treeThoughtsPositioned: TreeThoughtPositioned[]) => {
+  // Calculate isGreaterThought at LayoutTree level
+  const cursorIsGreaterThanPrevSibling = useSelector(state => {
+    try {
```

Comment:
It's bad practice to use try-catch for control flow. Very bad performance implications.

Reply:
I've removed the `try..catch` block, thanks for the insight.